UI/GamepadEmu.h

```diff
@@ -283,6 +283,7 @@ namespace CustomKeyData {
 		ImageID i; // UI ImageID
 		uint32_t c; // Key code
 	};
+	// NOTE: This list can be freely reordered, we store the keycode, not the index.
```

Comment:
Are you sure about this one? AFAIR the saved setting is a bitmask as it can support multiple ones. There is an assert to be sure we have less than 64 keys in this array. Sadly the result of a short sighted implementation on my side, I totally underestimated the need for reordering...

Reply:
Oh yeah, I think you're right... I'll change this comment and move it before merge. We should probably store a list of keycodes instead...